tools/gendocs/gen_openshift_docs.go

```diff
@@ -51,12 +53,16 @@ func main() {
 	out := os.Stdout
 	cmd := cli.NewOcCommand(&bytes.Buffer{}, out, ioutil.Discard)
 
-	outFile := filepath.Join(outDir, "oc-by-example-content.adoc")
+	fileName := "oc-by-example-content.adoc"
```

Comment:
Should this be "microshift-oc-by-example.adoc"?

Reply:
The prefix is added down on lines 60-62, so I think this is OK as a base filename.